src/settings/SettingsPage.js

```diff
@@ -63,15 +64,21 @@ export const SettingsPage = ({
         </div>
       </Box> }
 
-    <Box className='mb3 pa4-l pa2'>
-      <div className='lh-copy charcoal'>
+      <Box className='mb3 pa4-l pa2'>
+        <div className='lh-copy charcoal'>
         <Title>{t('app:terms.publicGateway')}</Title>
-        <Trans i18nKey='publicGatewayDescription' t={t}>
-          <p>Choose which <a className='link blue' href="http://docs.ipfs.tech/concepts/ipfs-gateway/#public-gateways" target='_blank' rel='noopener noreferrer'>public gateway</a> you want to use to open your files.</p>
-        </Trans>
-        <PublicGatewayForm/>
-      </div>
-    </Box>
+          <Trans i18nKey='publicSubdomainGatewayDescription' t={t}>
+            <p>Choose which <a className='link blue' href="https://docs.ipfs.tech/concepts/ipfs-gateway/#subdomain" target='_blank' rel='noopener noreferrer'>Subdomain Gateway</a> you want to use when generating shareable links (Default: Subdomain Isolation)</p>
```

Comment:
nit: this is GUI app, let's keep it short and avoid technical jargon. mentioning origin isolation can be skipped, we link to relevant docs already. 

mind updating here and in `public/locales/en`?

```suggestion
            <p>Select a default <a className='link blue' href='https://docs.ipfs.tech/concepts/ipfs-gateway/#subdomain' target='_blank' rel='noopener noreferrer'>Subdomain Gateway</a> for generating shareable links.</p>
```